providers/stripe.go

```diff
@@ -0,0 +1,29 @@
+package providers
+
+const Stripe Provider = "stripe"
+
+func init() {
+	// Stripe configuration
+	SetInfo(Stripe, ProviderInfo{
+		AuthType: ApiKey,
+		BaseURL:  "https://api.stripe.com",
+		ApiKeyOpts: &ApiKeyOpts{
+			Type:        InHeader, // Can also be InQuery
+			HeaderName:  "Authorization",
+			ValuePrefix: "Bearer ",
+			DocsURL:     "https://api.6sense.com/docs/#get-your-api-token",
```

Comment:
@mohammedfuta2000 This is for another provider, please update it to `https://docs.stripe.com/keys`

Reply:
corrected